src/walberla_bridge/src/lattice_boltzmann/LBWalberlaImpl.hpp

```diff
@@ -695,6 +752,101 @@ class LBWalberlaImpl : public LBWalberlaBase {
     }
   }
 
+  [[nodiscard]] bool is_gpu() const noexcept override {
+#if defined(__CUDACC__)
+    return true;
```

Comment:
Should this really be true unconditionally or actually depend on the arch template param?

Reply:
Good point.